src/util.c

```diff
@@ -709,6 +709,37 @@ dup_over_stdio(int *fdp){
 	return ret;
 }
 
+/**
+ * Generated random bytes.
+ *
+ * \param size Number of bytes to be generated.
+ *
+ * \return \c Newly allocated buffer on success, NULL otherwise.
+ **/
+uint8_t *
+get_random_bytes(uint num) {
```

Comment:
@sameo Seems like this is  new system call introduced in 3.17, would not be available in older kernels. The system call itself was added very recently in glibc: https://lwn.net/Articles/711013/.
I think reading from /dev/urandom is fairly standard way to get random data.